Sources/ThirdPartyMailer/ThirdPartyMailer.swift

```diff
@@ -67,3 +67,21 @@ open class ThirdPartyMailer {
         application.open(url, options: [:], completionHandler: completion)
     }
 }
+
+#if swift(>=5.5.2)
+    @available(iOS 13, *)
+    @MainActor
+    extension ThirdPartyMailer {
+        open class func openCompose(_ client: ThirdPartyMailClient = .systemDefault, recipient: String? = nil, subject: String? = nil, body: String? = nil, cc: String? = nil, bcc: String? = nil, with application: UIApplication) async -> Bool {
+            return await withCheckedContinuation { continuation in
+                openCompose(client, recipient: recipient, subject: subject, body: body, cc: cc, bcc: bcc, with: application) { result in
+                    continuation.resume(returning: result)
+                }
+            }
+        }
+
+        open class func openCompose(_ client: ThirdPartyMailClient = .systemDefault, recipient: String? = nil, subject: String? = nil, body: String? = nil, cc: String? = nil, bcc: String? = nil) async -> Bool {
```

Comment:
Couldn’t we define just one method, with a default value for the `application` parameter?

Reply:
Good question!

I tried `@MainActor` here but still could not add this default parameter:
<img width="1165" alt="image" src="https://user-images.githubusercontent.com/2396817/150983270-55bc4b40-ee6a-46b8-9c17-3632815ac1b4.png">